.github/workflows/third_party_scan.yml

```diff
@@ -2,6 +2,8 @@ name: Third party deps scan
 on:
   # Only the default branch is supported.
   branch_protection_rule:
+  push:
+    branches: [ main ]
```

Comment:
Would this be non-blocker?

Reply:
Yes this is non blocking because of `fail-on-vuln: false`

Reply:
Here's the documentation for that flag https://github.com/google/osv-scanner/blob/main/.github/workflows/osv-scanner-reusable-pr.yml#L41